react/src/components/flag/SprkFlag.js

```diff
@@ -4,8 +4,10 @@ import classnames from 'classnames';
 
```

Comment:
Let's add the TODOs somewhere in this file to deprecate as well

Reply:
![image](https://user-images.githubusercontent.com/4342363/89425179-ad91a380-d706-11ea-9612-f5bc3e414d79.png)
Done!